src/webpage/src/stats/stats_frequency.rs

```diff
@@ -0,0 +1,73 @@
+use std::sync::Arc;
+
+use egui::mutex::Mutex;
+use serde::{Deserialize, Serialize};
+use url::Url;
+use web_sys::window;
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Frequency {
+    pub frequency: f32,
+}
+
+pub fn get_stats_frequency(stats_frequency: &Arc<Mutex<f32>>) {
+    let location = window().unwrap().location();
+    let host = location.host().unwrap();
+    let protocol = location.protocol().unwrap();
+    let url = Url::parse(&format!("{protocol}//{host}/stats/frequency"))
+        .unwrap()
+        .to_string();
+
+    let mut request = ehttp::Request::get(url);
+    request.headers.insert("Content-Type", "application/json");
+
+    let stats_frequency = stats_frequency.clone();
+    ehttp::fetch(
+        request,
+        move |result: ehttp::Result<ehttp::Response>| match result {
+            Ok(response) => {
+                if let Some(json) = response.text() {
+                    if let Ok(Frequency { frequency }) = serde_json::from_str(json) {
+                        *stats_frequency.lock() = frequency;
+                    }
+                }
```

Comment:
```suggestion
                if let Ok(Frequency { frequency }) = response.json() {
                    *stats_frequency.lock() = frequency;
                }
```

Reply:
Any problem on this approach ?

Reply:
no method `.json()` found in `ehttp::Response`

Reply:
oh, I'll check the `json` feature flag

Reply:
yep, that's it, I'll update =D